internal/pkg/manifest/workload.go

```diff
@@ -328,3 +329,10 @@ func dockerfileBuildRequired(workloadType string, svc interface{}) (bool, error)
 	}
 	return required, nil
 }
+
+func stringP(s string) *string {
+	if s == "" {
```

Comment:
hmm is this necessary does `{{if .ImageConfig.Location}}` evaluate to true if the pointer points to an empty string?

Reply:
yeah it is necessary. otherwise i tried and the result is like this:
```yaml
image:
  # Docker build arguments. You can specify additional overrides here. Supported: dockerfile, context, args.
  build: Dockerfile
  # The Docker image used for your service.
  location: 
  # Port exposed through your container to route traffic to it.
  port: 5000
```